### PR TITLE
Defer & minor fixes

### PR DESCRIPTION
**Pull request by twokul**

### CODE REVIEW

File: addon/components/ui-autocomplete.js [hunk 1]
```diff
@@ -83,14 +84,6 @@ export default Component.extend(OptionListAriaMixin, {
     }
   }),
 
-  value: computed({
```
Comment (pzuraq) [inline addon/components/ui-autocomplete.js]: I know this doesn't work, but if we don't watch value somehow then we don't have data-down. If something changes the value externally it won't propagate. Perhaps an observer?

File: addon/components/ui-autocomplete.js [hunk 2]
```diff
@@ -57,9 +58,9 @@ export default Component.extend(OptionListAriaMixin, {
 
     @return {Boolean} Whether promise is pending or not
   */
-  isPromisePending: computed('proxyContent.isPending', 'value', {
+  isPromisePending: computed('proxyContent.isPending', 'query', {
     get() {
-      const value = this.get('value');
+      const value = this.get('query');
```
Comment (bl4ckm0r3) [inline addon/components/ui-autocomplete.js]: change `value` for consistency?

File: addon/components/ui-autocomplete.js (hunk 1 above)
Comment (twokul) [inline addon/components/ui-autocomplete.js]: right.

File: addon/components/ui-autocomplete.js (hunk 2 above)
Comment (twokul) [inline addon/components/ui-autocomplete.js]: :+1:

**Pull request merged**